src/harness/harness.ts

```diff
@@ -1028,7 +1037,16 @@ module Harness {
 
                         case 'newline':
                         case 'newlines':
-                            newLine = setting.value;
+                            if (setting.value.toLowerCase() === 'crlf') {
+                                options.newLine = ts.NewLineKind.CarriageReturnLineFeed;
+                            } else if (setting.value.toLowerCase() === 'lf') {
+                                options.newLine = ts.NewLineKind.LineFeed;
+                            } else if (setting.value === '\\n') {

```

Comment:
A grep search revealed only the contextualTyping.ts uses this other @newline flag as in the comment, no matches for @newlines plural.
// Handle old usage, e.g. contextualTyping.ts:// @newline: \n

I could use newlineflag for this new option, or change the old seldom-used option to something more internal, e.g. @normalizeNewline since it seems to be used in normalizeLineEndings(...).